feat(docs): Update docs and images for auto-assignment (#9291)
getsentry/sentry-docs: docs/product/issues/ownership-rules/index.mdx

The general format of a rule is: `type:pattern owners`

 - **`type`** - Can be either `path`, `module`, `url`, or `tags.TAG_NAME`, which matches the specified tag.
 - **`pattern`** - The pattern you're matching on, which supports unix-style [glob syntax](<https://en.wikipedia.org/wiki/Glob_(programming)>). For example, add `*` to match anything and `?` to match a single character. _This is not a regex._ For example:
   - `src/javascript/*` for a `path` type
   - `https://www.example.io/checkout` for a `url` type

>    - `Chrome 81.0.*` for a `tags.browser` type

Reviewer: Could you add an additional example showing how to use `?` and maybe for a `module` type?

>  - **`owners`** - The owner, or list of owners, identified by either the email of the Sentry user or the name of the team, prefixed with `#`. Thus, the owner may be `example@company.com` or `example1@company.com example2@company.com` or `#backend-team`. A space separates each owner in the list of owners.

Reviewer: ```suggestion
 - **`owners`** - The owner or a list of owners, separated by a space. Owners can be identified by either the Sentry user's email or the name of the team, prefixed with `#`. For example:
   - `example@company.com`
   - `example1@company.com example2@company.com` 
   - `#backend-team`
```

### Suggested Assignees

> On the **Issue Details** page, you'll see suggested assignees based on ownership rules matching the event, and [suspect commits](/product/releases/suspect-commits/). You can manually assign the issue to a suggested assignee by selecting an option on the dropdown. An event can have multiple suggested assignees if it matches multiple ownership rules.

Reviewer: ```suggestion
On the **Issue Details** page, you'll see suggested assignees based on ownership rules matching the event, and [suspect commits](/product/releases/suspect-commits/). You can manually assign the issue to a suggested assignee by selecting one from the dropdown. An event can have multiple suggested assignees if it matches multiple ownership rules.
```
